src/PrestaShopBundle/Entity/Repository/RequestSqlRepository.php

```diff
@@ -112,10 +130,15 @@ public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria = n
      *
      * @param SearchCriteriaInterface $criteria
      *
+     * @deprecated deprecated since 1.7.8.0
+     * @see RequestSqlQueryBuilder::buildQueryBySearchCriteria
+     *
      * @return QueryBuilder
      */
     private function buildQueryBySearchCriteria(SearchCriteriaInterface $criteria)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error(
            sprintf(
                'The "%s()" method is deprecated since 1.7.8.0. Use %s instead.', 
                __METHOD__, 
                RequestSqlQueryBuilder::class . '::buildQueryBySearchCriteria()'
            ), 
            E_USER_DEPRECATED
        );
```